test/test-v8debugapi.ts

```diff
@@ -409,8 +419,10 @@ describe('v8debugapi', function() {
           assert.ifError(err);
           api.wait(bp, function(err) {
             assert.ifError(err);
-            api.clear(bp);
-            done();
+            api.clear(bp, function(err) {
```

Comment:
Nit:  The  linter should have marked this `err` as shadowing the outer `err`, and required the inner `err` to have a different name (I usually use `err1` and `err2`).  I'm curious why the linter isn't flagging this.

Reply:
as in the whole file, such as where api.set and then api.wait, the code uses only 'err', it looks like linter would't complain for this. I will not change for now and we can follow up on this issue.